### DISCUSSION TITLE
Gofr env Configurations

### DISCUSSION THREAD

**Discussion by surajit-zs** [gofr-dev/gofr#429] [Q&A] [answered]
how do we set different env configs for a service for different environments?
I was going through this https://gofr.dev/docs/quick-start/configuration

so, If I want to run my application with dev.env locally then what is the process

**Comment by aryanmehrotra** [comment, marked as answer]
You can set your system env `APP_ENV` to `dev` and have env file name in the following format `.dev.env`.

We will update the same in documentation, it is being tracked here - https://github.com/gofr-dev/gofr/issues/434
